### ISSUE TITLE
Can the method be applied to scenes?

### ISSUE THREAD

**Issue by imoinuddin** [sxyu/svox2#57]
Fantastic work on this ! 

I wanted to know if the method can be applied towards 3d scene reconstructions as well ? (Example: for re-creation of an indoor space / room)

**Comment by sarafridov** [comment]
Absolutely! You can see an example of this in our video at around 4:30: https://alexyu.net/plenoxels/

**Comment by yjcaimeow** [comment]
Can the method be applied to outdoor scenes? For example KITTI with 1km scene.
